modules/apps/asset/asset-taglib/npmscripts.config.js

```diff
@@ -14,6 +14,7 @@
 
 module.exports = {
 	federation: {
+		bridges: [],
```

Comment:
Again, super confusing 😂

Reply:
I added `bridges: true` to remove that in one place of `npm-scripts` but then tried to use it and it failed in another :cry: . So I'll try to fix it in `npm-scripts` and change all references in `liferay-portal` in the next PR.

Still, `bridges: true` doesn't look much better... :roll_eyes:

Reply:
> Still, bridges: true doesn't look much better... 🙄

From someone that might not know what it does, `bridges: true` indicates, "yeah, do bridges, whatever", but `bridges: []` feels more like "nah, don't create any bridges, don't you see this empty array?".

Reply:
I'll make sure it works for the next npm-scripts release, then :+1: